src/pubd/commands.rs

```diff
@@ -1,30 +1,36 @@
 use std::fmt;
 
-use crate::commons::eventsourcing::CommandDetails;
-use crate::commons::eventsourcing::SentCommand;
-use crate::commons::remote::rfc8183;
-use crate::commons::{
-    actor::Actor,
-    api::{PublishDelta, PublisherHandle, RepositoryHandle, StorableRepositoryCommand},
+use rpki::uri;
+
+use crate::{
+    commons::{
+        actor::Actor,
+        api::{PublisherHandle, RepositoryHandle, StorableRepositoryCommand},
+        eventsourcing::{CommandDetails, SentCommand},
+        remote::rfc8183,
+    },
+    pubd::PubdEvt,
 };
-use crate::pubd::Evt;
 
 //------------ Cmd ---------------------------------------------------------
 pub type Cmd = SentCommand<CmdDet>;
 
 //------------ CmdDet ------------------------------------------------------
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 #[allow(clippy::large_enum_variant)]
-#[serde(rename_all = "snake_case")]
+#[serde(rename_all = "snake_case", tag = "type")]
 pub enum CmdDet {
-    AddPublisher(rfc8183::PublisherRequest),
-    RemovePublisher(PublisherHandle),
-    Publish(PublisherHandle, PublishDelta),
-    SessionReset,
```

Comment:
Is removal of session reset a loss of functioanilty or is it no longer relevant?